assets/templates/header-component.html

```diff
@@ -167,6 +167,27 @@
                           comprar
                         </button>
                       </div>
+                      <div class="user-plan--type">
+                        <p><strong>498</strong> Suscriptores disponibles</p>
+                        <div class="dp-request-sent">
+                          <button
+                            class="user-plan close-user--menu"
+                            type="button"
+                          >
+                            solicitud enviada
```

Comment:
This text is only text or label, maybe we need to add css for <p> or <label>, see https://makingsense.atlassian.net/wiki/spaces/DOP/pages/1390051366/Flujo+de+Contrataci+n+de+Plan+Junio+2021

Reply:
I did not understand the question.
I'll call you and we'll see it if you want. 👍